test/integration/api/RebalancingAPI.spec.ts

```diff
@@ -1058,12 +1058,36 @@ describe('RebalancingAPI', () => {
           DEFAULT_CONSTANT_AUCTION_PRICE
         );
 
+        const expectedTokenAddresses = await rebalancingSetToken.getCombinedTokenArray.callAsync();
+
+        const expectedInflowAddressArray = expectedTokenFlowArrays['inflow'].reduce((acc, unit, index) => {
```

Comment:
Depending on your test case, I think it's more effective to spell out what the expected values are for your particular set up parameters instead of recreating the same logic here which would always produce a passing test. e.g. `expectedOutflowAddressArray = [ { ... } ]`